lxd/instance/drivers/driver_qemu.go

```diff
@@ -6857,6 +6857,39 @@ func (d *qemu) migrateSendLive(pool storagePools.Pool, clusterMoveSourceName str
 		return err
 	}
 
+	// Derive the effective storage project name from the instance config's project.
+	storageProjectName, err := project.StorageVolumeProject(d.state.DB.Cluster, d.project.Name, dbCluster.StoragePoolVolumeTypeCustom)
+	if err != nil {
+		return err
+	}
+
+	// Notify the shared disks that they're going to be accessed from another system.
+	for _, dev := range d.expandedDevices.Sorted() {
+		if dev.Config["type"] != "disk" || dev.Config["path"] == "/" {
+			continue
+		}
+
+		// Load the pool for the disk.
+		diskPool, err := storagePools.LoadByName(d.state, dev.Config["pool"])
```

Comment:
An optimisation here would be to store the diskPool in a map keyed by name, so that if there are multiple disks from the same pool you dont have to load the pool from DB multiple times.